src/database/migrations/20211008033500_cities.ts

```diff
@@ -0,0 +1,18 @@
+import { Knex } from 'knex'
+
+export async function up (knex: Knex): Promise<void> {
+  return knex.schema.hasTable('cities').then(function (exists) {
+    if (!exists) {
+      return knex.schema.createTable('cities', function (table) {
+        table.string('id', 5).primary()
+        table.string('name', 60).notNullable()
+        table.boolean('is_active').notNullable().index()
+        table.string('area_id', 2).index()
```

Comment:
area_id is foreign key from areas table?
why area_id length is 2 while area_id in areas table length is 4 ?
CMIIW

Reply:
iam forgot thanks